DisCatSharp.Docs/articles/misc/hosting.md

```diff
@@ -22,6 +22,65 @@ Any modern hardware will work just fine, new or used.
 
 Depending on how complex your bot is, you may even consider purchasing a Raspberry Pi ($35).
 
+# Termux
+
+For those who don't have some PC or other hardware lying around alternatively you could use your phone. Using [Termux](https://termux.dev/en/) and a package called [proot-distro](https://github.com/termux/proot-distro), which we use to install a Debian vm and setup DotNET to run the bot. To those interested the steps are given below:
+
+## Requirements
+
+- An Android 5+
+- Termux
+- An internet connection
+
+## Setup
+
+- Initialize Termux
+```sh
+pkg update && pkg upgrade -y
+```
+*Note, it might ask you for input, just click enter and let the default option execute*
+
+- Install proot-distro package
+```sh
+pkg install proot-distro -y
+```
+
+- Install a Ubuntu vm
+```sh
+proot-distro install ubuntu -y
+```
+*Note, it may take a bit depending on your internet speed*
+
+- Login into Ubuntu and initialize Ubuntu
+```sh
+proot-distro login ubuntu
+```
+```sh
+apt-get update -y && apt-get upgrade -y
+```
+*Note, it may take a bit depending on your internet speed*
+
+- Update and install git and sdk
+```sh
+apt-get update -y && apt-get install -y dotnet-sdk-8.0 git -y
```

Comment:
I think, in most cases, actually compiling the bot on a mobile device wouldn't be the best idea so maybe just downloading the runtime is enough. Whether you want to change this is up to you, just a thought.

Reply:
Alright, will look into that, thanks for feedback